llvm/lib/Target/AMDGPU/AMDGPUTargetMachine.cpp

```diff
@@ -337,6 +337,10 @@ static cl::opt<bool> EnableStructurizerWorkarounds(
     cl::desc("Enable workarounds for the StructurizeCFG pass"), cl::init(true),
     cl::Hidden);
 
+static cl::opt<bool> EnableSwLowerLDS("amdgpu-enable-sw-lower-lds",
+                                      cl::desc("Enable sw lower lds pass"),
+                                      cl::init(true), cl::Hidden);
```

Comment:
The name is misleading now that it's not a software lowering pass. At least the description should mention sanitizers?